src/Congo.WebApi/Data/CartAccess/GetCartsHandler.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Congo.WebApi.Data.Models;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+
+namespace Congo.WebApi.Data.CartAccess
+{
+    public class GetCartsHandler : IRequestHandler<GetCartsQuery, IEnumerable<Cart>>
```

Comment:
We can remove the GetCarts feature since the user will never have more than 1 cart, it's persisted across all devices since we are saving it in the db